broker/broker-route-manager/src/main/java/org/codice/ddf/broker/route/manager/DynamicRouteDeployer.java

```diff
@@ -32,15 +32,23 @@
 public class DynamicRouteDeployer implements ArtifactInstaller {
   private static final Logger LOGGER = LoggerFactory.getLogger(DynamicRouteDeployer.class);
 
-  private final CamelContext camelContext;
+  /**
+   * This map should be static in order to maintain a reference to the existing map entries after
+   * a restart of this bundle. This is because a new instance of the DynamicRouteDeployer is
+   * instantiated when the bundle restarts, so the reference to the existing map entries are
+   * cleaned up, therefore this map should be static to maintain the existing reference.
```

Comment:
Could you mention that install is not called on bundle restart to process the RouteDefinitions again so we need to make this static so that it survives the instance being destroyed and recreated during a bundle restart.

Reply:
Thanks, that is more descriptive. Fixed.